packages/subgraph/tests/admin/utils.ts

```diff
@@ -96,3 +97,80 @@ export function createProposalExecutedEvent(
 
   return createProposalExecutedEvent;
 }
+
+export function createGrantedEvent(
```

Comment:
I'd rather follow the current structure and have a util.ts and dedicated event generation functions in tests

Reply:
but these 2 functions will always be the same for every plugin when they need it. Maybe put it in `dao's` utils ?

Reply:
I am still the opinion we should not intermangle the plugins and the core even more in the subgraph. It will be already pretty hard to unmangle and this is why I would leave code duplicaiton